localisation/000_wc_military_localization.csv

```diff
@@ -1,17 +1,13 @@
 #CODE;ENGLISH;FRENCH;GERMAN;;SPANISH;;;;;;;;;x
 #Military Technologies;;;;;;;;;;;;;;x
-TECH_INFANTRY;Conventional Warfare;;Konventionelle Kriegsf�hrung;;;;;;;;;;;x
-TECH_CAVALRY;Specialized Warfare;;Irregul�re Kriegsf�hrung;;;;;;;;;;;x
-TECH_SKIRMISH;Ley Attunement;;Ley Einstimmung;;;;;;;;;;;x
-TECH_MELEE;Tactics Development;;Taktikentwicklung;;;;;;;;;;;x
+TECH_INFANTRY;Frontline Troops;;;;;;;;;;;;;x
+TECH_CAVALRY;Charging Troops;;;;;;;;;;;;;x
```

Comment:
If we are using this route, perhaps `Shock Troops` instead?

Reply:
Shocks Troops are already taken. The name of Aerial Infantry and Cavalry group.

Reply:
I don't understand the conflict, isn't this just naming a technology? Or does this show up in modifier tooltips?

Reply:
It sounds like this tech buffs only Shock Troops, while it also gives bonus to Large Infantry.

Reply:
How about then `Assault Troops`?

Reply:
Sounds fine!